commons/src/main/java/org/eclipse/kapua/commons/util/KapuaFileUtils.java

```diff
@@ -79,6 +79,7 @@ public static File getAsFile(String filePath) throws KapuaSettingException {
 
             if (hasHttpScheme(filePath)) {
                 // FIXME:
+                // After fixing this method, needs to adjust/fix the tests
```

Comment:
Which is the issue with this method?

Reply:
@Coduz 
That’s the comment I found in that class. So if that method ever changes or whatever ... it is necessary to adjust the test. That's all.